inspirehep/dojson/common/base.py

```diff
@@ -216,10 +217,13 @@ def collections(self, key, value):
 
     def get_value(value):
```

Comment:
Yeah, self-contained rules read better, and a slowdown of 2 microseconds per call is very acceptable. BTW, @kaplun, this is more evidence that DoJSON's trouble is on their side...